blueocean-analytics-tools/src/main/resources/index.jelly

```diff
@@ -0,0 +1,4 @@
+<?jelly escape-by-default='true'?>
+<div>
+  This plugin is a part of Blue Ocean UI
```

Comment:
Its just a placeholder to keep plugin system happy. Guess readme describes what the plugin really is anyways.